### DISCUSSION TITLE
displacy don't show any graph

### DISCUSSION THREAD

**Discussion by RGreyWolf** [explosion/spaCy#12516] [Help: Best practices]
I want to see dep graph result but don't show any result
This is my code
```
import spacy
from spacy import displacy

nlp = spacy.load("en_core_web_sm")

doc = nlp("I want to see dep graph.")
displacy.serve(doc, style = "dep")  
```

**Comment by svlandeg** [comment]
Hi! Can you paste the console output, what do you see? It should include a link to where you can view the visualisation.

**Reply by RGreyWolf** [reply]
Thanks for your response and I'm sharing console output.


```
Using the 'dep' visualizer
Serving on http://0.0.0.0:5000 ...
```

And I've tried for http://127.0.0.1.5000 but, it doesn't work.

**Reply by svlandeg** [reply]
What about http://127.0.0.1:5000/?

**Reply by RGreyWolf** [reply]
oh, dear, That's nice
it works for me
Thanks very much

**Reply by RGreyWolf** [reply]
> What about http://127.0.0.1:5000/?

how are they different?
